frontend/src/app/main/ui/components/forms.cljs

```diff
@@ -412,3 +411,36 @@
                                                  "caution" (:caution item))}
             [:span.text (:text item)]
             [:span.icon {:on-click #(remove-item! item)} i/cross]]])])]))
+
+;; --- Validators
+
+(defn all-spaces?
+  [value]
+  (let [trimmed (str/trim value)]
+    (str/empty? trimmed)))
+
+(def max-length-allowed 250)
+
+(def max-uri-length-allowed 2048)
+
+(defn max-length?
+  [value length]
+  (> (count value) length))
+
+(defn validate-length
+  [field errors-msg length]
+  (fn [errors data]
+    (cond-> errors
+      (max-length? (get data field) length)
+      (assoc field {:message errors-msg}))))
+
+(defn validate-spaces
+  [fields error-msg]
```

Comment:
The same here, I think you should receive a single field and use the validator multiple times